kubetools/kubernetes/config/job.py

```diff
@@ -56,6 +56,10 @@ def make_job_config(
         {'KUBE_JOB_ID': job_id},
     )
 
+    # if resources exist in job config, use it here
+    if config.get('resources'):
+        resources = config.get('resources')
```

Comment:
I believe it would do the exact same as this
```suggestion
    # if resources exist in job config, use it here
    resources = config.get('resources')
```
Then, the function argument can be removed and the variable inlined since it is only used in 1 place

Reply:
[9387f5d](https://github.com/EDITD/kubetools/pull/134/commits/9387f5d3f7d1b716aa1c60e3145846c33f398e77)

Reply:
Think empty dict is what we want to return here over `None`? Not sure `None` fits the k8s job spec as well

Reply:
None is (was?) the default value of the function argument

Reply:
> Then, the function argument can be removed and the variable inlined since it is only used in 1 place

Still pending

Reply:
Yeh, `None` could work but I think `{}` is more inline with what I would expect to see if I saw a job_config with no resources defined.

Done the second part now!
[fe17c2e](https://github.com/EDITD/kubetools/pull/134/commits/fe17c2e0ca4984f8bee291b52669db87f93679dc)